src/Common/src/Interop/Unix/libssl/Interop.libssl.cs

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class libssl
+    {
+
+        [DllImport(Interop.Libraries.LibSsl, EntryPoint = "ERR_load_BIO_strings")]
```

Comment:
I'm surprised this doesn't get an EntryPointNotFoundException, since it's a function in libcrypto, not libssl.

It's also not necessary to call; since CryptoInitializer would have already loaded it.  Oh, and it isn't called... that explains a lot (other than why it's here).